packages/dotcom-ui-shell/src/components/GTMBody.tsx

```diff
@@ -7,10 +7,14 @@ const GTMBody = ({ flags }: { flags: TFlagsData }) => {
     return null
   }
 
+  const src = flags['ads-first-party-gtm']
+    ? 'https://www.ft.com/page-resources/'
```

Comment:
**question**: is there a reason `GTMHead` uses a relative URL and this uses absolute?

Reply:
None whatsoever... thanks for catching!